.eslintrc.js

```diff
@@ -13,6 +13,7 @@ module.exports = {
     /** Turn off strict enforcement */
     "@typescript-eslint/ban-types": "off",
     "@typescript-eslint/ban-ts-comment": "off",
+    "@typescript-eslint/consistent-type-imports": "error",
```

Comment:
if you want to implement this later, this line is the only required change.

Then run `yarn lint-fix` and `yarn format` and `npx changeset`.